src/Microsoft.DotNet.XHarness.Apple/AppRunnerBase.cs

```diff
@@ -52,7 +52,7 @@ public abstract class AppRunnerBase
         protected async Task<string> FindDevice(TestTargetOs target)
         {
             IHardwareDevice? companionDevice = null;
-            IHardwareDevice device = await _hardwareDeviceLoader.FindDevice(target.Platform.ToRunMode(), _mainLog, includeLocked: false, force: false);
```

Comment:
why do you remove this please?

Reply:
I noticed it was not doing anything in the implementation so it's probably a leftover from the old XHarness code